.env.template

```diff
@@ -1,3 +1,10 @@
 APP_ENV="prod"
 APP_PORT="8080"
-DATABASE_URL="postgres://chai:@localhost:5432/chai?sslmode=disable"
+DB_USER="chai"
+DB_PASS=""
+DB_HOST="localhost:5432"
+DB_PORT="5432"
+DB_NAME="chai"
+DATABASE_URL="postgres://${DB_USER}:${DB_PASS}@${DB_HOST}/${DB_NAME}?sslmode=disable"
```

Comment:
Can we keep the old `DATABASE_URL` variable in there, and remove the `DB_*` and `DATABASE_URL` on line 8? I don't forsee these values changing during dev, especially if we're using the Nix shell.